loadgenerator/loadgen

```diff
@@ -458,6 +496,13 @@ def _get_parser():
             description='Show the status of all load generator jobs whether running or not')
     subparser.set_defaults(func=status)
 
+    subparser = subparsers.add_parser('autorun',
+            help='Create load generator with Cloud Run',
+            description='automatically starts a load generation job for a target address with Cloud Run')
+    subparser.add_argument('target_address', type=str,
+            help='The target address (without http/s prefix) to send laod to')
+    subparser.set_defaults(func=run_auto)
```

Comment:
Can you think of a way to test this? Maybe write a pytest like Nick's or Levi's?

Reply:
Yeah definitely. Test the parser or test the autorun function?

Reply:
Both could be useful. I know we're getting short on time, so whatever you think would be highest impact. It would be nice if we could get some testing in at least though

Reply:
For sure!